examples/src/main/java/com/expediagroup/sdk/lodgingconnectivity/SandboxDataManagementClientUsageExample.java

```diff
@@ -71,24 +70,24 @@ public static void main(String[] args) {
         deletePropertyIfExists();
 
         //  ******* Create Property *******
-        CreatePropertyInput createPropertyInput = CreatePropertyInput
-                .builder()
-                .name(Optional.of(PROPERTY_NAME))
+        CreatePropertyInput createPropertyInput = new CreatePropertyInput
+                .Builder()
```

Comment:
Let's create a follow-up ticket to support the `CreatePropertyInput.builder()` method style (for all input objects).

Reply:
Done!